helper_test.go

```diff
@@ -58,6 +58,16 @@ func TestGenWebpAbs(t *testing.T) {
 	}
 }
 
+func TestGenEtag(t *testing.T) {
+	var data = "./pics/png.jpg"
+	var expected = "W/\"1020764-262C0329\""
+	var result = GenEtag(data)
+
+	if result != expected {
```

Comment:
use `assert.NotEqual`, document for [assert](https://godoc.org/github.com/stretchr/testify/assert)